.github/workflows/master-push.yml

```
@@ -107,3 +76,36 @@ jobs:
         run: |
           echo "${{ secrets.DOCKERHUB_PASSWORD }}" | docker login --username rvdockerhub --password-stdin
           docker image push ${TAG}
+
+          
+  cut-release:
+    name: 'Cut Release'
+    runs-on: ubuntu-latest
+    needs: [nix-cache, dockerhub]
```

Until the `macos-13` Nix cache job is fixed, I think we should just go with

```suggestion
    needs: dockerhub
```

Otherwise, we won't be able to make a release.